src/jit/codegenarmarch.cpp

```diff
@@ -649,10 +650,16 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
 
                 structSize = varDsc->lvSize(); // This yields the roundUp size, but that is fine
                                                // as that is how much stack is allocated for this LclVar
-                isHfa      = varDsc->lvIsHfa();
+                isHfa = varDsc->lvIsHfa();
+#ifdef _TARGET_ARM64_
+                gcPtrs     = gcPtrArray;
                 gcPtrCount = varDsc->lvStructGcCount;
                 for (unsigned i = 0; i < gcPtrCount; ++i)
                     gcPtrs[i]   = varDsc->lvGcLayout[i];
+#else // _TARGET_ARM_
+                gcPtrs     = treeNode->gtGcPtrs;
+                gcPtrCount = treeNode->gtNumSlots;
+#endif // _TARGET_ARM_
             }
```

Comment:
`treeNode->gtGcPtrs` and `treeNode->gtNumSlots` can be used when `FEATURE_PUT_STRUCT_ARG_STK` is set, but ARM64 is not use this feature.